apps/web/src/screens/Profile/components/TransactionHistory/TransactionDetails.tsx

```diff
@@ -67,7 +67,7 @@ const TransactionDetails: FC<Transaction> = ({
 							]}
 						/>
 						<Text style={styles.tokenName}>
-							{amount} {token.metadata?.symbol}
+							{amount} {token.metadata?.name?.replaceAll('\u0000', '').trim()}
```

Comment:
seems duplicated with `apps/web/src/screens/Profile/components/TransactionHistory/HistoryItem.tsx`, any chance we want a shared util method?

Reply:
> seems duplicated with `apps/web/src/screens/Profile/components/TransactionHistory/HistoryItem.tsx`, any chance we want a shared util method?

@cloudle  i got the token name from `getSolanaMetadata`, so i think if we don't want to duplicate this code, we should update `getSolanaMetadata`